packages/grid/x-data-grid-pro/src/components/GridRowReorderCell.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { unstable_composeClasses as composeClasses } from '@mui/material';
+import {
+  GridEvents,
+  GridRenderCellParams,
+  GridRowEventLookup,
+  gridRowTreeDepthSelector,
+  gridSortModelSelector,
+  useGridApiContext,
+  useGridSelector,
+  getDataGridUtilityClass,
+  gridEditRowsStateSelector,
+} from '@mui/x-data-grid';
+import { isObjectEmpty } from '@mui/x-data-grid/utils/utils';
+import { DataGridProProcessedProps } from '../models/dataGridProProps';
+import { useGridRootProps } from '../hooks/utils/useGridRootProps';
+
+type OwnerState = {
+  classes?: DataGridProProcessedProps['classes'];
+  isDraggable: boolean;
+};
+
+const useUtilityClasses = (ownerState: OwnerState) => {
+  const { isDraggable, classes } = ownerState;
+
+  const slots = {
+    root: ['rowReorderCell', isDraggable && 'rowReorderCell--draggable'],
+    container: ['rowDraggableContainer'],
```

Comment:
It's not very a "container". Maybe something with "placeholder", `rowReorderCellPlaceholder`?